projects/plugins/jetpack/extensions/blocks/blogging-prompts/settings.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Blogging prompt settings.
+ *
+ * @since $$next-version$$
+ *
+ * @package automattic/jetpack
+ */
+
+namespace Automattic\Jetpack\Extensions\BloggingPrompts\Settings;
+
+use Automattic\Jetpack\Status\Host;
+
+/**
+ * Renders the settings field for enabling/disabling blogging prompts in the editor.
+ *
+ * @return void
+ */
+function enabled_field_callback() {
+	$option  = jetpack_are_blogging_prompts_enabled();
+	$checked = ( $option ? 'checked' : '' ); ?>
+<input name="jetpack_blogging_prompts_enabled" id="jetpack_blogging_prompts_enabled" type="checkbox" value="1"<?php echo esc_attr( $checked ); ?> />
+<p id="jetpack-blogging-prompts-enabled-description" class="description"><?php esc_html_e( 'Displays a writing prompt when starting a new post.', 'jetpack' ); ?></p>
```

Comment:
```suggestion
<label><input name="jetpack_blogging_prompts_enabled" id="jetpack_blogging_prompts_enabled" type="checkbox" value="1"<?php echo esc_attr( $checked ); ?> />
<span id="jetpack-blogging-prompts-enabled-description" class="description"><?php esc_html_e( 'Displays a writing prompt when starting a new post.', 'jetpack' ); ?></span></label>
```

Reply:
I was using the section title to the left as the label, since there's only one setting, but I think you're right, it's better with the label inline. Updated.